tests/unit/AddNameTranslation.spec.ts

```diff
@@ -241,5 +241,7 @@ describe('Add Name Translation component', () => {
     await translationInput.setValue('Lower case will be capitalized')
 
     expect(wrapper.find(addTranslationInput).element.value).toBe('LOWER CASE WILL BE CAPITALIZED')
+
+    wrapper.destroy()
```

Comment:
Added to fix memory leak in tests (and also in other test suites below).

Before this, the tests would hang when 75% done. With this (and the others below), the tests run fine to the end.